modules/idImportLibrary.js

```diff
@@ -257,6 +261,10 @@ export function setConfig(config) {
     _logError('The required url is not configured');
     return;
   }
+  if (!isActivityAllowed(ACTIVITY_ENRICH_UFPD, activityParams(MODULE_TYPE_UID, 'idImportLibrary'))) {
```

Comment:
this should be MODULE_TYPE_PREBID. All other types represent a relationship with a 3rd party vendor which changes how consent has to be gathered in some jurisdictions.